### ISSUE TITLE
The training process stops right after epoch 13 and can't be continued. 

### ISSUE THREAD

**Issue by HuitingLiu** [OpenNMT/OpenNMT#49]
Here is the output of continue cmd:

```th train.lua -data data/demo-train.t7 -save_model model  -train_from model_epoch13_214.48.t7 -save_every 20   -gpuid 4 -continue 
Loading checkpoint 'model_epoch13_214.48.t7'...	
 * vocabulary size: source = 50004; target = 50004	
 * additional features: source = 0; target = 0	
 * maximum sequence length: source = 50; target = 51	
 * number of training sentences: 145525	
 * maximum batch size: 64	
Building model...	
Initializing parameters...	
Resuming training from epoch 14 at iteration 1...	
Loading data from 'data/demo-train.t7'...	
 * number of parameters: 84814004	
Preparing memory optimization...	
 * sharing 69% of output/gradInput tensors memory between clones	
Start training...	
 * vocabulary size: source = 50004; target = 50004	
 * additional features: source = 0; target = 0	
 * maximum sequence length: source = 50; target = 51	
 * number of training sentences: 145525	
 * maximum batch size: 64	
Building model...	
Initializing parameters...	
 * number of parameters: 84814004	
Preparing memory optimization...	
 * sharing 69% of output/gradInput tensors memory between clones	
Start training...	
[1]+  Done                    th train.lua -data data/demo-train.t7 -save_model model -train_from model_epoch13_214.48.t7 -save_every 20 -gpuid 4 -continue```

**Comment by srush** [comment]
It looks like the culprit is this line 

` opt.epochs = checkpoint.options.epochs`
https://github.com/OpenNMT/OpenNMT/blob/master/train.lua#L460

Is your expected behavior to use `--continue` but to respecify the number of epochs? 

One workaround is to not use continue, but to instead give the full set of command line arguments that you want to start from (learning rate etc.). A hack to just remove line 460, and give -epochs

**Comment by HuitingLiu** [comment]
Thanks. The hack solved the problem.
